src/components/DropList/DropList.test.js

```diff
@@ -841,6 +867,12 @@ describe('Selection', () => {
         )
       ).toBeTruthy()
       expect(onSelect).toHaveBeenCalledWith(itemToSelect, itemToSelect)
+      expect(onListItemSelectEventSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          event: expect.objectContaining({ type: 'click' }),
+          listItemNode: getByText(itemToSelect.label).parentElement,
+        })
+      )
```

Comment:
Not to this, line best relates to tests :) Should we add some test case to cover Enter/Space keydown as well? Especially in Select version it's a new code that I believe would be worth testing

Reply:
Good idea, I just added them